### PR TITLE
Feature/hrelease v6

### PR DESCRIPTION
**Pull request by hmcts-shajahan**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: Jenkinsfile_CNP [hunk 1]
```diff
@@ -167,29 +169,30 @@ withPipeline(type, product, component) {
 
   onPR {
     env.ENVIRONMENT = "preview"
-    env.CASE_API_URL = "http://privatelaw-hearings-api-pr-${CHANGE_ID}.service.core-compute-preview.internal"
+    env.CASE_API_URL = "https://fis-hmc-api-pr-${CHANGE_ID}.service.core-compute-preview.internal"
+    env.CASE_API_TEST_URL = "https://fis-hmc-api-pr-${CHANGE_ID}.service.core-compute-preview.internal"
```
Comment (hmcts-kodatala) [inline Jenkinsfile_CNP:173]: env.CASE_API_TEST_URL. need. to add onMaster ,demo,...

**Review by hmcts-kodatala — COMMENTED** [review]
pls check review comments

**Review by hmcts-kodatala — APPROVED** [review]
OK

File: Jenkinsfile_CNP (hunk 1 above)
Comment (hmcts-shajahan) [reply Jenkinsfile_CNP:173]: done

**Pull request merged**